examples/experimental/terrain/src/terrain-layer/TerrainLayer.js

```diff
@@ -0,0 +1,261 @@
+import GL from "@luma.gl/constants";
+// import { Layer } from "@deck.gl/core";
+import { Model, Texture2D } from "@luma.gl/core";
+import {picking, project32, gouraudLighting} from '@deck.gl/core';
+import { BitmapLayer } from "@deck.gl/layers";
+import vs from "./terrain-layer-vertex";
+import fs from "./terrain-layer-fragment";
+import SquareGridGeometry from "./square-grid-geometry";
+
+const DEFAULT_TEXTURE_PARAMETERS = {
+  [GL.TEXTURE_MIN_FILTER]: GL.LINEAR_MIPMAP_LINEAR,
+  [GL.TEXTURE_MAG_FILTER]: GL.LINEAR,
+  [GL.TEXTURE_WRAP_S]: GL.CLAMP_TO_EDGE,
+  [GL.TEXTURE_WRAP_T]: GL.CLAMP_TO_EDGE
+};
+
+const defaultProps = {
+  // ...BitmapLayer.defaultProps,
+  images: { type: "object", value: [], async: true },
+  cutoffHeightM: { type: "number", value: 40.0 },
```

Comment:
This seems rather arbitrary. Use `0`?

Reply:
Sounds good. 40 is a little above sea level - cutting out the ocean emphasized the land nicely in an example, but isn't a good default.